ansible/veos_vtb

```diff
@@ -17,6 +17,7 @@ all:
           - t1-8-lag
           - t1-64-lag-clet
           - t1-backend
+          - t1-smartswitch
```

Comment:
How to support other t1 topo like t1-8-lag, t1-28-lag, t1-32-lag?

Reply:
Define port channel in topo files. 

When I raised this PR, we were focusing on vtestbed and VPP image. LAG isn't supported on sonic-vpp yet.

Reply:
@zjswhhh could you consider renaming this topo accordingly to reflect that this is a vtestbed? We might need to reserve "t1-smartswitch" for the actual physical smartswitch testbeds.

Reply:
Hi @zjswhhh, sure separate PR is fine. The reason I suggested this is there are some upstream commits that assume "t1-smartswitch" topo name for generating default config for physical smartswitch. 

For eg: https://github.com/sonic-net/sonic-buildimage/pull/17326/

There is also another commit by Cisco #15153  which added a smartswitch-t1 topo recently. I am trying to get them to change it to "t1-smartswitch" to be consistent everywhere.